### ISSUE TITLE
Please add support for comma separated interface versions

### ISSUE THREAD

**Issue by zeptognome** [BigWigsMods/packager#164]
Recent clients (10.2.7, and 4.4.0) are able to support ##Interface lines with comma separated versions.

The current version of BWP (2.2.2) is unable to handle this syntax fully.
my toc line:
"## Interface: 100206, 40400, 100207, 110000, 11502"

Relevant lines from my GHA:
/home/runner/work/_actions/BigWigsMods/packager/v2/release.sh: line 1220: printf: 100206,40400,100207,110000,1: invalid number

Packaging BetterBags_ImportExport
Current version: v1.0.9
Previous version: v1.0.6
Build type: retail non-alpha non-debug
Game version: 100206.15.2

WARNING: No CurseForge game version match for "100206.15.2", using "10.2.6"

WARNING: No Wago game version match for "100206.15.2", using "10.2.7"

parse error: Objects must consist of key:value pairs at line 1, column 202
There was an error creating release.json

The zip package was created successfully, only the version routing seemed to generate errors.

Thanks for your time

**Comment by zeptognome** [comment]
> my main concern was people starting to use it without more game support then being confused

and yes, I realize that 11502 doesn't yet support csv and will be ignored, but technically 
`## Interface: 100206, 40400, 100207, 110000` is valid today

**Comment by nebularg** [comment]
> and yes, I realize that 11502 doesn't yet support csv and will be ignored, but technically `## Interface: 100206, 40400, 100207, 110000` is valid today

that's false, 10.2.6 doesn't support it, 10.2.7 will

**Comment by zeptognome** [comment]
Following up with some partially successful results

Interface: 11502, 40400, 100207, 110000
and
uses: BigWigsMods/packager@toc-csv

With the exception that a warning was given for 110000, as I expected, GHA successful deployed on Github and Curse. WowInterface not tested (yet)

BWP reported that Wago API didn't recognize 10.2.7 and I had to manually change the tag on the wago website for retail from 10.2.6 to 10.2.7. Wago did pickup Cata Classic and Classic SoD versions correctly. I published a different addon without a csv toc an hour earlier and BWP was able to match 10.2.7 on Wago

Relevant lines from the workflow:
Build type: multi-version non-alpha non-debug
Game version: 11.0.0,10.2.7,4.4.0,1.15.2
WARNING: No CurseForge game version match for "11.0.0", using "10.2.7"
Uploading BetterBags_ImportExport-v1.0.10.zip (10.2.7,4.4.0,1.15.2 release) to https://www.curseforge.com/projects/1010211
WARNING: No Wago game version match for "10.2.7", using "10.2.6"
WARNING: No Wago game version match for "11.0.0", using "10.2.6"
Uploading BetterBags_ImportExport-v1.0.10.zip (10.2.6,4.4.0,1.15.2 release) to https://addons.wago.io/addons/rNkPdw6a
